doc/release-notes/0.44/0.44.md

```diff
@@ -0,0 +1,177 @@
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/19173">#19173</a></td>
+<td valign="top">Flag names that refer to the fields of <tt>J9BuildFlags</tt> is changed in the Direct Dump Reader (DDR) code</td>
+<td valign="top">All versions</td>
+<td valign="top">User plug-ins in the DDR code might contain code that has <tt>J9BuildFlags</tt> reference to read the build flags in the system dump data. When the <tt>J9BuildFlags</tt> is generated for each build, the raw name of the build flags might be used in the plug-in code rather than the <tt>cname</tt> that appears in <tt>j9cfg.h</tt> (derived from <tt>j9cfg.h.ft1</tt>). Therefore, mapping of flag names in the plug-in code with the names that are specified in <tt>j9cfg.h</tt> was inconsistent and because of this inconsistency, some flag tests might not apply to all system dumps.
+
+To reduce complexity and improve the uniformity of how the DDR code uses the flags, you must use the names that are specified in <tt>j9cfg.h</tt> as flag names in the plug-in code.
+</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/19045">#19045, <a href="https://github.com/eclipse-openj9/openj9/issues/19069">#19069,</a></td>
+<td valign="top">VMID query in the <tt>jcmd</tt> tool enhanced</td>
+<td valign="top">All versions</td>
+<td valign="top">Earlier in OpenJ9, when sending a <tt>jcmd</tt> command to a VM, you had to run <tt>jcmd -l</tt> to retrieve all the pids for all the VMs found on the machine. Then, you had to use <tt>jcmd [vmid] [command]</tt> to send the command to the specific VM.
+
+For OpenJDK compatibility, OpenJ9 now supports direct use of the Java process name, full or partial, as the ID to send the <tt>jcmd</tt> command.
+
+The <tt>jcmd</tt> tool also now supports specifying <tt>0</tt> as a VMID to target all VMs.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/18720">#18720</a>, <a href="https://github.com/eclipse-openj9/openj9/issues/18996">#18996</a></td>
+<td valign="top">A new system property, <tt>-Dcom.ibm.tools.attach.fileAccessUpdateTime</tt>, is added to prevent automatic deletion of the Attach API control files in the <tt>/tmp</tt> folder.</td>
+<td valign="top">All versions (Linux&reg;)</td>
+<td valign="top">In Linux environments, by default <tt>systemd-tmpfiles</tt> automatically deletes all files and directories that are stored in the <tt>/tmp/</tt> folder that have not been changed or read within a specific time period. This deletion of files creates a problem for long running VM processes, such as Attach API, that need the control files in the <tt>/tmp/</tt> folder to operate.
+
+You can prevent the automatic deletion of the Attach API control files in the <tt>/tmp</tt> folder by setting the <tt>-Dcom.ibm.tools.attach.fileAccessUpdateTime</tt> system property.</td>
```

Comment:
This should mention it's set to 8 days by default, so you don't actually have to set the property unless the default needs to be changed.